account_due_list_days_overdue/models/account_overdue_term.py

```diff
@@ -4,9 +4,9 @@
 # © 2016 Therp BV (<http://therp.nl>).
```

Comment:
@mreficent Can you please modify the Copyrights as per new [standards](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L2)

`# Copyright 2016 Therp BV (<http://therp.nl>).`